combustion-android-ble/src/main/java/inc/combustion/framework/ble/uart/meatnet/NodeReadFeatureFlagsRequest.kt

```diff
@@ -0,0 +1,26 @@
+package inc.combustion.framework.ble.uart.meatnet
+
+import inc.combustion.framework.ble.putLittleEndianUInt32At
+
+internal class NodeReadFeatureFlagsRequest(
+    serialNumber: String,
+    requestId: UInt? = null
+) : NodeRequest(populatePayload(serialNumber), NodeMessageType.GET_FEATURE_FLAGS, requestId)
```

Comment:
This class was a little tricky for me to understand.  But I think it is because we're writing over the 4 byte serial number of the probe endpoint with the 10 byte serial number of the node.

A couple of things might make it easier to understand.  move `populatePayload` to `NodeRequest` and call it something like `populateDirectPayload`.  I think maybe we'll want that anyway if all direct messages need a serial number.

Reply:
Nevermind -- I was misreading -- I though `populatePayload` was a method on `NodeRequest`, and that (probe) serial number was part of message header for all node message, not part of the payload.  What's here makes sense.